common/buildings/00_terrain_specific_buildings.txt

```diff
@@ -293,6 +293,9 @@ pastures_07 = {
 	character_modifier = {
```

Comment:
The innovation needs its own icon.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>